README.md

```diff
@@ -1,2 +1,555 @@
-# cli
-Clean and elegant CLI development kit.
+# {Name}
+
+[![Build](https://github.com/candy-kingdom/cli/workflows/Build/badge.svg)](https://github.com/JoshuaLight/chalice-restful/actions)
+
+_Clean and elegant CLI development kit._
+
+## Overview
+
+{Name} is a microframework that allows building declarative and nice-looking CLI apps.
+
+Consider the following simple program as a demonstration of {Name} capabilities:
+```py
+# say.py
+
+from {pyname} import cli, Arg, Flag, Opt
+
+@cli
+def say(phrase: Arg('a phrase to print'),
```

Comment:
In the [say.py](https://github.com/candy-kingdom/cli/blob/develop/examples/say.py) example I used comments that start with a capital letter and end with a period. I think that we should use a single style in both `examples` and README. Which style do you prefer?

Reply:
I used such a style because it's already used for the `--help` flag by the `argparse`, so I'd prefer to make the style consistent across all arguments.

Reply:
For me, this question is complicated, because I feel like the whole `argparse` format is not that good and should be changed as well. For example, `usage:` should be changed to `Usage:`, etc.

As for now, I'd prefer to stick with the defaults and to change things later if needed. What do you think?

Reply:
That's a valid point! Let's stick with the default format for now then. Do you want me to change descriptions in the `examples` folder to match this format?

Reply:
Yep, that would be nice of you!